src/licensedcode/data/rules/tsl-2018.RULE

```diff
@@ -0,0 +1,3 @@
+TIMESCALE LICENSE AGREEMENT
+
+Posted Date:  December 19, 2018
```

Comment:
You can likely eschew having this date in a rule

Reply:
... You can likely eschew having this date in a rule ... unless you saw this notice in the wild? and is it worth it detect the older 2018 versiom?